libs/block-editor/src/lib/extensions/services/dot-image/dot-image.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { uploadBinaryFile } from '@dotcms/utils';
+import { from, Observable } from 'rxjs';
+import { pluck, switchMap } from 'rxjs/operators';
+import { DotCMSTempFile } from '@dotcms/dotcms-models';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable()
+export class DotImageService {
+    constructor(private http: HttpClient) {}
+
+    get(data: File | File[], maxSize?: string): Observable<any> {
```

Comment:
Doc.

This is not a `get` is more like an `publishContent` or something like that.

Also type.